requirements/requirements_development_local.txt

```diff
@@ -1,4 +1,5 @@
 -r https://raw.githubusercontent.com/OpenSlides/openslides-datastore-service/${DATASTORE_COMMIT_HASH}/requirements/requirements-general.txt
--r partial/requirements_datastore_locally.txt
+-r https://raw.githubusercontent.com/OpenSlides/openslides-auth-service/${AUTH_COMMIT_HASH}/auth/libraries/pip-auth/requirements.txt
```

Comment:
this is not local, it is from github

Reply:
changed it to relative paths

Reply:
I take it back, it does not work as simply as I though. The problem is that the datastore and auth service only get mounted via `docker-compose` _after_ the container has finished building. Since the requirements files are needed during build time, we cannot simply access them. I suppose this is also the reason why I did it the way it currently is. We could think about changing this, but not in the scope of this PR. Also the requirements change very seldom, so for most cases it is okay to use the github requirements file.